.claude/commands/dev/git-status.md

```
@@ -0,0 +1,39 @@
+# Git Status Command
+
+Show detailed git repository status
+
+*Command originally created by IndyDevDan (YouTube: https://www.youtube.com/@indydevdan) / DislerH (GitHub: https://github.com/disler)*
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)